.github/CONTRIBUTING.md

```diff
@@ -109,28 +109,65 @@ Code coverage is monitored for every PR and for the code base as a whole using [
 
 ### Running the Tests
 
```

Comment:
Might be good to update the "Prerequisites for running the tests" section to mention the minimum versions, i.e. Python 3.12 and mitmproxy 11.0.2.